WISHLIST.md

```diff
@@ -0,0 +1,64 @@
+WISHLIST to make LUYA a better CMS
+==============
+
+Simply put your suggestions and wishes for improvements in this basic text file to help us to make LUYA your favorite futur CMS.
+
+
+Installation
+-------------------
+
+- integrate composer fxp/composer-asset-plugin installation and availibilty test in composer create project.
+
+
+
+Console
+-------------------
+- beautify console appearance and shorten luya console commands (tab complete doesn't work OSX)
```

Comment:
should be possible http://php.net/readline_completion_function - but should be a pull request for the yii2 framework